index.html

```diff
@@ -0,0 +1,47 @@
+<DOCTYPE! html>
+  <html>
+    <head>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    </head>
+    <body>
+      <nav>

```

Comment:
It looks like you're using the <nav> tag for the ad sidebar, and you left off the yellow <nav> in the mockup.